nbs/examples/Forecasting_TFT.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>It would be better to have the release version and just install it with <code>pip install neuralforecast</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>